concrete_datastore/api/v1/filters.py

```diff
@@ -22,6 +20,48 @@
 )
 
 
+def get_filter_field_type(model_class, param):
+    """
+    Return the type of the target field that we want to filter
+    """
+    splitted_param = param.split('__')
+
+    if len(splitted_param) == 1:
+        return model_class._meta.get_field(
+            splitted_param[0]
+        ).get_internal_type()
+    elif len(splitted_param) == 2:
+        first_param, second_param = splitted_param
+        if (
+            model_class._meta.get_field(first_param).get_internal_type()
+            != 'ForeignKey'
+        ):
+            raise ValidationError(
+                {
+                    "message": (
+                        f"{param}: Multi level filters"
+                        " available only for foreign keys"
+                    )
+                }
+            )
+        return (
+            model_class._meta.get_field(first_param)
+            .remote_field.model._meta.get_field(second_param)
+            .get_internal_type()
+        )
+
+    # If we have field__fkfield__fkfieldfkfield or more, raise an error
+    # We can at most have field__fkfield as filter
+    else:
+        raise ValidationError(
+            {
+                "message": (
+                    f"{param}: filter not available for more than 2 levels"
```

Comment:
if there is a constraints on a 2-dimension after split, that mean a "1 level"... No ?

Reply:
Yes it is one level, not two